.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,93 @@
+name: "Helm Charts"
+
+# This workflow provides two things:
+# On explicit dispatch, it packages, uploads and releases the chart to the
+# charts repo.
+#   It uses the appVersion and version as present in Chart.yaml.
+# On push to master, it packages and uploads a helm chart for the pushed commit.
+#   It updates both appVersion and version to be specific to this commit.
+#   See tools/release/helm.sh#dev_version
+# This workflow can be tested by any owner with a fork of kumahq/kuma and kumahq/charts
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release charts
+        required: false
+        default: false
+        type: boolean
+
+env:
+  CR_VERSION: 1.3.0
+
+  GH_USER: "github-actions[bot]"
+  GH_EMAIL: "<41898282+github-actions[bot]@users.noreply.github.com>"
+  GH_OWNER: ${{ github.repository_owner }}
+
+jobs:
+  package:
+    name: Package
+    runs-on: ubuntu-latest
+    outputs:
+      filename: ${{ steps.package.outputs.filename }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Seems like you need to make this branch specific no?

Reply:
The action checks out whatever ref triggered the workflow